bin/upgrading/import-includes.pl

```diff
@@ -40,23 +39,30 @@ BEGIN
     next unless -f $path;
 
     # now get filetime
-    my $ftimedisk = (stat($path))[9];
-    my $ftimedb = $dbh->selectrow_array("SELECT updatetime
-                    FROM includetext WHERE incname=?", undef, $file)+0;
-    
+    my $ftimedisk = ( stat($path) )[9];
+    my $ftimedb   = $dbh->selectrow_array(
+        "SELECT updatetime
+                    FROM includetext WHERE incname=?", undef, $file
```

Comment:
The linebreak and indenting inside this string doesn't make sense now that the beginning of the string has been outdented